packages/sdk-core/src/proving/proving-manager.ts

```diff
@@ -1,10 +1,23 @@
 // Copyright 2022 @webb-tools/
 // SPDX-License-Identifier: Apache-2.0
 
+import type { NoteProtocol } from '@webb-tools/wasm-utils';
+
 import { ProvingManagerSetupInput, ProvingManagerWrapper } from '@webb-tools/sdk-core/proving/proving-manager-thread.js';
-import { Proof } from '@webb-tools/wasm-utils';
 
-export type ProofI = Omit<Proof, 'free'>;
+type VAnchorProof = {
+  readonly inputUtxos: Array<any>;
```

Comment:
Is there no way to type these arrays? Can we not use `Utxo` type?

Reply:
Handled types here